Documentation/guides.xml

```diff
@@ -4,6 +4,7 @@
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation="https://www.phpdoc.org/guides vendor/phpdocumentor/guides-cli/resources/schema/guides.xsd"
 >
+    <project title="Render guides" />
```

Comment:
I assume, this is doubled by accident?

Reply:
Yes, accidentall commit sadly. Reverted that part. Thanks for catching this :)